site/docs/v1/tech/admin-guide/cli.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: doc
+title: FusionAuth CLI
+description: Learn about the FusionAuth admin CLI tool
+---
+
+The FusionAuth command line interface (CLI) tool allows you to manipulate FusionAuth from the command line.
+
+## Prerequisites
+
+The CLI tool requires node. It's tested with version 19 but should work with modern versions of node.
+
+## Installation
+
+You can install this using `npm`.
+
+```
+npm i -g @fusionauth/cli
+```
+
+
+Then you can run commands.
+
+```
+fusionauth -k <APIKEY> theme:download <themeid>
```

Comment:
@mooreds Trying to go through example.  I get a "unknown option -k" here. I tried the --help but only see -V and -h switches.  Am I missing something?

Reply:
Yes, I think I put the args in the wrong order. Just pushed updated doco that should work.